#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("resolution", double_t, 0, "resolution of diff", 0.03, 0.0, 10.0)
gen.add("noise_filter", int_t, 0, "noise filter val", 4, 1, 100)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "OctreeChangePublisher"))
